client/src/stores/qrcode.js

```diff
@@ -0,0 +1,5 @@
+import { store } from "react-stax";
+
+const qrcode = store({ value: "No value yet!" });
```

Comment:
`qrcode` isn't really a good name because the NFC reader will have the work into this very same store, maybe name it `inspector`?